html-skeleton/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<head>
+  <title>HTML Skeletons</title>
+  <body>
+    Hello, HTML!
+  </body>
+</head>
```

Comment:
You are missing the `html` element, and `body` should be a sibling to `head`, not a child of it.